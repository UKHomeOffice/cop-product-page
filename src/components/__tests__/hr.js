import React from "react"
import renderer from "react-test-renderer"
import Hr from "../hr";

describe("Hr", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Hr/>)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })

  it("renders correctly in content", () => {
    const tree = renderer
      .create(<Hr inContent={true}/>)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})