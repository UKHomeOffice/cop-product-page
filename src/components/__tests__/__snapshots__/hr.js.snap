// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hr renders correctly 1`] = `
<div
  className="gov-hr"
/>
`;

exports[`Hr renders correctly in content 1`] = `
<div
  className="gov-hr govuk-width-container"
/>
`;
